roles/install_ocp/meta/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+galaxy_info:
+  role_name: redhat_cop.tower_utilities.ocp_install
+  author: Chris Renwick
+  description: Role to install Ansible Tower on Openshift
+  company: Red Hat
+
+  license: license (MIT)
+
+  min_ansible_version: 2.7
+
+  platforms:
+    - name: EL
+      versions:
+        - 7
+
+  galaxy_tags:
+    - linux
+    - system
+    - ansible
+
+dependencies:
+  - role: redhat_cop.tower_utilities.pre_tasks
+    vars:
+      tower_releases_url: "{{ tower_releases_url }}"
```

Comment:
So this doesnt seem to work (my bad).

Probably the best solution is to add a roles section directly into `tasks/main.yml`. 

```
roles:
  - redhat_cop.tower_utilities.pre_tasks
```